feature/profile/src/main/java/com/iteneum/profile/Profile.kt

```diff
@@ -1,4 +1,10 @@
-package com.iteneum
+package com.iteneum.profile
+
+/**
+ * Class that contains [Profile] data fields.
+ *
+ * @author Jose Miguel Garcia Reyes
+ */
 
 data class Profile (
     val profileImage : String = "",
```

Comment:
You can name Profile properties without profile at the begining, for example instead of **profileImage** you can name it as **image** only.

Reply:
Move this file to domain layer, create a domain layer and put it there.

Reply:
Profile properties renamed and file relocated!